### PR TITLE
Faithlife.Logos.Beta

### PR DESCRIPTION
**Pull request by JasonCook599**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41187)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
```
Enter a pull request number: 41187

Manifest validation failed.
Manifest Error: The multi file manifest has inconsistent field values. Field: PackageIdentifier Value: Faithlife.Logos File: Faithlife.Logos.Beta.locale.en-US.yaml
Manifest Error: The multi file manifest has inconsistent field values. Field: PackageVersion Value: 9.10.17 File: Faithlife.Logos.Beta.locale.en-US.yaml

Press any key to continue . . .
```

File: manifests/f/Faithlife/Logos/Beta/9.11.3/Faithlife.Logos.Beta.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+PackageIdentifier: Faithlife.Logos.Beta
+PackageVersion: 9.11.3
+PackageLocale: en-US
+Publisher: Faithlife, LLC
+PublisherUrl: https://www.logos.com/
+PublisherSupportUrl: https://support.logos.com/
+PrivacyUrl: https://faithlife.com/privacy
+Author: Faithlife, LLC
+PackageName: Logos Bible Software
```
[the remark this answers is not in the document]
Comment (ItzLevvie) [reply manifests/f/Faithlife/Logos/Beta/9.11.3/Faithlife.Logos.Beta.locale.en-US.yaml:12]: to make sure it doesn't conflict with the stable package.

**Comment by JasonCook599** [comment]
> ```
> Enter a pull request number: 41187
> 
> Manifest validation failed.
> Manifest Error: The multi file manifest has inconsistent field values. Field: PackageIdentifier Value: Faithlife.Logos File: Faithlife.Logos.Beta.locale.en-US.yaml
> Manifest Error: The multi file manifest has inconsistent field values. Field: PackageVersion Value: 9.10.17 File: Faithlife.Logos.Beta.locale.en-US.yaml
> 
> Press any key to continue . . .
> ```

Fixed. Didn't update when copying metadata from the stable version.

File: manifests/f/Faithlife/Logos/Beta/9.11.3/Faithlife.Logos.Beta.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: Faithlife.Logos.Beta
+PackageVersion: 9.11.3
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://downloads.logoscdn.com/LBS9/InstallerBeta/9.11.0.0003/Logos-x64.msi
+  InstallerSha256: DC574389FA4643F4439FA9393A2D158AD0C01B5651415498E28456FD67768F78
+  ProductCode: '{B107AC4F-0CFB-4C1B-BC0A-BCC1886C0E91}'
```
Comment (ItzLevvie) [inline manifests/f/Faithlife/Logos/Beta/9.11.3/Faithlife.Logos.Beta.installer.yaml:12]:
```suggestion
  ProductCode: '{B107AC4F-0CFB-4C1B-BC0A-BCC1886C0E91}'
  Scope: user
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Looks like the pipelines crashed for this PR.

Please close and re-open the PR to re-run the pipelines.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**